### ISSUE TITLE
Go to definition not working in the decompiled class

### ISSUE THREAD

**Issue by testforstephen** [dgileadi/vscode-java-decompiler#8]
When open a Java file in VS Code, and click some API from a library class, the decompiler is able to decompile the .class file. This is good feature.

But in the opened .class file, click the method invocation in the decompiled code, nothing happens. If supporting "Go to definition" in the decompiled code, that will help a lot.

**Comment by dgileadi** [comment]
Agreed. Unfortunately this feature depends on RedHat's Java support; it isn't something that this extension can do by itself.
